crypto/fipsmodule/ec/p521.c

```diff
@@ -220,7 +216,13 @@ static void p521_felem_cmovznz(p521_limb_t out[P521_NLIMBS],
 
 // NOTE: the input and output are in little-endian representation.
 static void p521_from_generic(p521_felem out, const EC_FELEM *in) {
+#ifdef OPENSSL_BIG_ENDIAN
+  uint8_t tmp[66];
+  bn_words_to_little_endian(tmp, 66, in->words, EC_MAX_WORDS);
```

Comment:
```suggestion
  bn_words_to_little_endian(tmp, 66, in->words, P521_NLIMBS);
```

Reply:
it's `EC_MAX_WORDS` because `EC_FELEM` object needs to be able hold field element for any curve that we implement, but each individual curve has it's own field element length.